src/clr-angular/utils/drag-and-drop/droppable/droppable.ts

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Directive, ElementRef, EventEmitter, Input, OnDestroy, OnInit, Output, Renderer2} from "@angular/core";
+import {Subscription} from "rxjs/Subscription";
+
+import {DomAdapter} from "../../dom-adapter/dom-adapter";
+import {ClrDragEventExt} from "../drag-event-external";
+import {ClrDragEvent, ClrDragEventType} from "../interfaces/drag-event";
+import {ClrDropTolerance} from "../interfaces/drop-tolerance";
+import {ClrDragAndDropEventBus} from "../providers/drag-and-drop-event-bus";
+
+@Directive({
+    selector: "[clrDroppable]",
+    providers: [DomAdapter],
+    host: {"[class.droppable]": "true", "[class.draggable-match]": "isDraggableMatch"}
+})
+export class ClrDroppable<T> implements OnInit, OnDestroy {
+    private dragStartSubscription: Subscription;
+    private dragMoveSubscription: Subscription;
+    private dragEndSubscription: Subscription;
+
+    private droppableEl: any;
+    private clientRect: ClientRect;
+
+    constructor(private el: ElementRef, private eventBus: ClrDragAndDropEventBus<T>, private domAdapter: DomAdapter,
+                private renderer: Renderer2) {
+        this.droppableEl = this.el.nativeElement;
+    }
+
+    private isDraggableMatch: boolean = false;
+    private _isDraggableOver: boolean = false;
+
+    set isDraggableOver(value: boolean) {
+        // We need to add/remove this draggable-over class via Renderer2
+        // because isDraggableOver is set outside of NgZone.
+        if (value) {
+            this.renderer.addClass(this.droppableEl, "draggable-over");
+        } else {
+            this.renderer.removeClass(this.droppableEl, "draggable-over");
+        }
+        this._isDraggableOver = value;
+    }
+
+    private _group: string|string[];
+    private _dropTolerance: ClrDropTolerance = {top: 0, right: 0, bottom: 0, left: 0};
+
+    @Input("clrGroup")
+    set group(value: string|string[]) {
+        this._group = value;
+    }
+
+    @Input("clrDropTolerance")
+    set dropTolerance(value: number|string|ClrDropTolerance) {
```

Comment:
@gnomeontherun this is the implementation I did as per our discussion on slack. All of the following options will work now.
```
[clrDropTolerance]="10"
clrDropTolerance="10"
clrDropTolerance="10"
clrDropTolerance="10 20"
clrDropTolerance="10 20 30"
```